tests/test_narrator.py

```diff
@@ -56,7 +62,7 @@ def test_off_air(self):
         MockAdapter.act.assert_not_called()
 
     def test_mic_cable_kinked(self):
-        MockAdapter = mock.MagicMock()
+        MockAdapter = mock.create_autospec(Adapter, instance=True)
```

Comment:
Seeing all these changes makes me feel like we should have a `get_mock_adapter` method here, too.